<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>258</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Símbolo monetario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>€</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>£</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>El símbolo aparecerá</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="afterRd">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Por ejemplo: 50€&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>después de la cifra.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="beforeRd">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Por ejemplo: $50&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>antes de la cifra.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>¿Notificar actualizaciones?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="updatesEnabled">
        <property name="text">
         <string>Notificar actualizaciones</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>Config</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
